### PR TITLE
Fix overflow issue with image aside on mobile

### PR DESCRIPTION
**Pull request by jmcolella**
@drewdrewthis @cisacke 👀  please

fixes issue on mobile

### CODE REVIEW

File: src/molecules/asides/image-aside/index.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ function ImageAside ( props ) {
             icon={ icon }
             wrapperClassName={ styles.icon } />
         :
-          <img src={ icon } className={ styles.icon } />
+          <div className={ styles.icon }>
+            <img src={ icon } />
+          </div>
       }
 
       <aside className={ styles.aside }>
```
Comment (drewdrewthis) [inline src/molecules/asides/image-aside/index.js]: I think you should update this to use the `TextComponent` -- we don't want this h5 hardwired in, and we'd probably want the TextComponent to handle the typography

Comment (jmcolella) [inline src/molecules/asides/image-aside/index.js]: 👍

**Comment by cisacke** [comment]
@jmcolella what's the issue this is fixing? I can't see anything wrong on master, but this also seems to work fine so LGTM! 🤷‍♀️

File: src/molecules/asides/ImageAside/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      }
+
+      <aside className={ styles.aside }>
+        <TextComponent
+          className={ styles.header }
+        >
+          { header }
+        </TextComponent>
+
+        { subheader && <TextComponent className={ styles.subheader }>{ subheader }</TextComponent> }
```
Comment (drewdrewthis) [inline src/molecules/asides/ImageAside/index.js]: I suggest breaking this up into multiple lines, but not a blocker.

**Pull request merged**